### ISSUE TITLE
raw_* content is nil

### ISSUE THREAD

**Issue by rememberlenny** [thoughtbot/griddler-sendgrid#9]
I am running Ruby 2.2.3 and unable to get sendgrid incoming parse to return raw_html information. The only response content I get is "body", but it is stripped of any formatting or HTML.

I believe the API for sendgrid might have changed since this gem was created. If this is correct, then it may need to be updated.

I didnt have this same problem with other services (Mandrill, Mailgun).

**Comment by sdhull** [comment]
Sometimes `raw_html` may not be present because not all clients send it. On the other hand, [if you have checked `Send Raw`](https://community.sendgrid.com/sendgrid/topics/what-do-the-inbound-parse-webhook-settings-actually-do?topic-reply-list%5Bsettings%5D%5Bfilter_by%5D=all&topic-reply-list%5Bsettings%5D%5Breply_id%5D=16871046#reply_16871046) in Sendgrid Parse options, it will not send *any* of the documented params and instead send a `_raw` param, thus completely breaking `griddler-sendgrid` and `griddler`.

tl;dr -- don't check "Send Raw" option. It will break all the things.

**Comment by stefannibrasil** [comment]
Closing this as this library has been deprecated: https://github.com/thoughtbot/griddler-sendgrid/pull/41
